kartothek/io/dask/delayed.py

```diff
@@ -439,6 +433,7 @@ def update_dataset_from_delayed(
     sort_partitions_by=None,
     secondary_indices=None,
     factory=None,
+    metadata_storage_format=DEFAULT_METADATA_STORAGE_FORMAT,
```

Comment:
We've been frequently discussing dropping this kwarg once and for all. I would suggest to not introduce it now for functions where it's been forgotten so far.